packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -1425,6 +1457,14 @@ function completeUnitOfWork(unitOfWork: Fiber): Fiber | null {
   return null;
 }
 
+function getRemainingExpirationTime(fiber: Fiber) {
+  const updateExpirationTime = fiber.expirationTime;
+  const childExpirationTime = fiber.childExpirationTime;
+  return updateExpirationTime > childExpirationTime
+    ? updateExpirationTime
+    : childExpirationTime;
```

Comment:
`Math.max`?

Reply:
We avoid using methods when there is syntax because it requires additional look ups where as a comparison can trivially be understood as a single op.